package.json

```diff
@@ -2,28 +2,32 @@
   "name": "server",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "src/index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "tsc -p tsconfig.json && nodemon dist/js/app.js"
+    "start": "node dist/index.js",
```

Comment:
I don't know how to make this work but I can't start the server this way

Reply:
Whoops... My bad for not updating you:
From now on, please run `npm run dev` instead of `npm start` while developing.

`ts-node-dev` is like `nodemon` , but runs on typescript without the need to compile! 🤩 

That's also why I deleted `nodemon`, and added two extra scripts: `build`, which...well... builds.... and `start`, which **after** the building of your code will run your compiled code as usual.

Don't worry, in a "perfect" world you'll only have to run `npm run dev`, and keep `npm start` to the OpenShift servers (where you don't have to worry about servers collapsing)

yay.